mintapi/api.py

```diff
@@ -760,6 +761,13 @@ def get_transactions_json(self, include_investment=False,
             start_date = datetime.strptime(start_date, '%m/%d/%y')
         except (TypeError, ValueError):
             start_date = None
+
+        # Converts the end date into datetime format - must be mm/dd/yy
+        try:
+            end_date = datetime.strptime(end_date, '%m/%d/%y')
+        except (TypeError, ValueError):
+            end_date = None
+
```

Comment:
Makes sense! My only thought here is that this is basically copy and pasted from above. Can this be abstracted into a little helper method and then you can simply call it twice with different dates?

Reply:
Yes, definitely can!  I will do that.